plugins/headlamp-plugin/bin/headlamp-plugin.js

```diff
@@ -269,24 +270,46 @@ function start() {
   }
 
   /**
-   * Inform if @kinvolk/headlamp-plugin is outdated.
+   * Inform if @kinvolk/headlamp-plugin is outdated, and check for updates if 24h have passed.
    */
-  function informIfOutdated() {
-    console.log('Checking if headlamp-plugin is up to date...');
-    const outdated = getNpmOutdated();
-    if ('@kinvolk/headlamp-plugin' in outdated) {
-      const url = `https://github.com/kinvolk/headlamp/releases`;
-      console.warn(
-        '    @kinvolk/headlamp-plugin is out of date. Run the following command to upgrade \n' +
-          `    See release notes here: ${url}` +
-          '    npx @kinvolk/headlamp-plugin upgrade'
+  async function informIfOutdated() {
+    const lastCheckTime = process.env.LAST_UPDATE_CHECK
+      ? parseInt(process.env.LAST_UPDATE_CHECK, 10)
+      : null;
```

Comment:
This logic means lastCheckTime can be set as null, and then on line 282 it's used in arythmetic.